src/cryptography/hazmat/backends/commoncrypto/ciphers.py

```diff
@@ -151,6 +160,8 @@ def __init__(self, backend, cipher, mode, operation):
         self.authenticate_additional_data(b"")
 
     def update(self, data):
+        self._plaintext_size_validator.update(data)
+        self._plaintext_size_validator.validate()
```

Comment:
I didn't think it was right for `validate` to increment a counter, such that calling `validate(data)` was not idempotent. At the same time, just calling `update` doesn't imply a validation occurs. `update_and_validate` is an acceptable compromise, though.